libpod/container_internal_common.go

```diff
@@ -1962,7 +1962,7 @@ func (c *Container) makeBindMounts() error {
 			case m.Destination == "/run/.containerenv":
 				hasRunContainerenv = true
 				break Loop
-			case m.Destination == "/run" && m.Source != "tmpfs":
+			case m.Destination == "/run" && m.Source != define.TypeTmpfs:
```

Comment:
not related to this change this must compare `m.type`

Reply:
@Luap99 im going to merge this ... do you wanna make a quick pr afterwards to fix this?